packages/lsb_release.rb

```diff
@@ -14,7 +14,8 @@ class Lsb_release < Package
   binary_sha256 ({
   })
 
-  depends_on 'help2man'
+  depends_on 'help2man' => :build
+  depends_on 'make' => :build
```

Comment:
Don't these both already get installed from buildessential which is necessary before any build?

Reply:
In theory? @esprout could you try a `crew update && crew install lsb_release` and see if that works?

Reply:
I see the issue here.  Since this doesn't have pre-built binaries, these need to be runtime dependencies since make needs to execute with `crew install` in addition to `crew build`.  The build process is specific to each system so pre-built binaries are not possible with this package.

Reply:
Changed these to runtime deps.